File: web/src/main/java/material/com/web/ui/WebContact.java
package material.com.web.ui;

import material.com.base.BasePresenter;
import material.com.base.BaseView;

/**
 * Created by zjl on 2017/4/13.
 */

public interface WebContact {
    interface Prenster extends BasePresenter{

    }

    interface View extends BaseView<Prenster>{
        void initUI();
    }
}

File: web/src/main/java/material/com/web/ui/WebContact.kt
package material.com.web.ui

import material.com.base.BasePresenter
import material.com.base.BaseView

/**
 * Created by zjl on 2017/4/13.
 */

interface WebContact {
    interface Prenster : BasePresenter

    interface View : BaseView<Prenster> {
        fun initUI()
    }
}
